packages/nns/src/errors/ledger.errors.ts

```diff
@@ -1,13 +1,13 @@
 import type { TransferError as RawTransferError } from "../../candid/ledger";
-import { ICP } from "../icp";
+import { Token } from "../token";
 import type { BlockHeight } from "../types/common";
 
 export class TransferError extends Error {}
 
 export class InvalidSenderError extends TransferError {}
 
 export class InsufficientFundsError extends TransferError {
-  constructor(public readonly balance: ICP) {
+  constructor(public readonly balance: Token) {
```

Comment:
a bit same discussion as we had before, if it's just the balance, then it isn't a Token since Token contains more information than just the balance. So to be accurate regarding this here I would use `bigint`.

Reply:
agree